core/src/test/java/org/infinispan/stats/SingleStatsTest.java

```diff
@@ -66,7 +67,8 @@ protected void configure(ConfigurationBuilder cfg) {
             // Off heap key/value size is 64 bytes
             size = UnpooledOffHeapMemoryAllocator.estimateSizeOverhead(size * 64);
             // Have to also include address count overhead
-            size += UnpooledOffHeapMemoryAllocator.estimateSizeOverhead(cfg.memory().addressCount() << 3);
+            // TODO: this may fail on CI due to having less CPUs - CONFIRM OR DENY before integrating
```

Comment:
Shouldn't matter now that INITIAL_SIZE is at least 256, or after you change it to *always* be 256 ;)